src/datasets/utils/tf_utils.py

```diff
@@ -285,7 +297,7 @@ def __iter__(self):
         with SharedMemoryContext() as shm_ctx:
             for i in range(num_workers):
                 worker_random_id = str(uuid4())
-                worker_name = f"datasets_tf_worker_{i}_{worker_random_id}"
+                worker_name = f"dw_{i}_{worker_random_id}"[:10]
```

Comment:
This is probably cleaner than the whole UUID, I agree

Reply:
I was eventually running into `OSError: [Errno 36] File name too long: ...` this is why I just stripped it to keep the first 10 characters